Bowling/Bowling/FrameNoBonusRollFactory.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class FrameNoBonusRollFactory
+    {
+        public const int totalPinsCount = 10;
```

Comment:
I would encourage you to go through this program and try to be consistent with how things are named.  In this class you have two public global variables.  One is Pascal case and the other is camel case.  This hinders the readability of the code because the way variables, functions, classes, etc. are named means something to the reader of the code.

Generally, Pascal case is reserved for class and function names.  Camel case is reserved for local variables and an underscore preceding a camel case string is reserved for global variables.

Further, is their a need for either of these variables to be `public`?  A good rule of thumb with regard to managing encapsulation is to make everything `private` until your hand is forced to make something with great-scope like `public`.  Public global variables is a code-smell and should almost always be avoided in classes with business logic.

Reply:
I went through all the code and I believe I resolved all casing issues.  I have removed the public variable and used a constructor to set this.